{"failedTests":[{"testName":"parallel/test-async-context-frame","timestamp":1725360895269,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win2022-COMPILED_BY-vs2022 #29880","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2022-COMPILED_BY-vs2022/29880/","buildNumber":29880,"builtOn":"test-rackspace-win2022_vs2022-x64-3","nodeVersion":"23.0.0","commitHash":"0ae2ae1e52aa70bdc876a9e414bb324dfb645388","callStack":[{"upstreamBuild":65441,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":73848,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":61848,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54526"}],"tap":{"ok":false,"name":"parallel/test-async-context-frame","id":109,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"▶ AsyncContextFrame\n  ✖ async-hooks\\test-async-local-storage-args.js (4855.5555ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-args.js failed with exit code 3221225477\n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Promise.all (index 0)\n        at async Suite.run (node:internal/test_runner/test:1268:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-async-await.js (4865.5722ms)\n  ✔ async-hooks\\test-async-local-storage-async-functions.js (4814.5434ms)\n  ✔ async-hooks\\test-async-local-storage-dgram.js (4965.5044ms)\n  ✔ async-hooks\\test-async-local-storage-enable-disable.js (4570.7848ms)\n  ✖ async-hooks\\test-async-local-storage-enter-with.js (4867.0779ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-enter-with.js failed with exit code 3221225477\n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Promise.all (index 5)\n        at async Suite.run (node:internal/test_runner/test:1268:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-errors.js (4709.4633ms)\n  ✔ async-hooks\\test-async-local-storage-gcable.js (4761.3555ms)\n  ✔ async-hooks\\test-async-local-storage-http-agent.js (5088.1007ms)\n  ✔ async-hooks\\test-async-local-storage-http.js (4470.027ms)\n  ✔ async-hooks\\test-async-local-storage-misc-stores.js (4503.1006ms)\n  ✔ async-hooks\\test-async-local-storage-nested.js (4796.1771ms)\n  ✔ async-hooks\\test-async-local-storage-no-mix-contexts.js (5278.4982ms)\n  ✔ async-hooks\\test-async-local-storage-promises.js (4927.463ms)\n  ✔ async-hooks\\test-async-local-storage-socket.js (4780.8599ms)\n  ✔ async-hooks\\test-async-local-storage-thenable.js (4864.6529ms)\n  ✔ async-hooks\\test-async-local-storage-tlssocket.js (4538.5307ms)\n  ✔ parallel\\test-async-local-storage-bind.js (4667.3036ms)\n  ✔ parallel\\test-async-local-storage-contexts.js (4696.3967ms)\n  ✔ parallel\\test-async-local-storage-deep-stack.js (4797.337ms)\n  ✔ parallel\\test-async-local-storage-exit-does-not-leak.js (4676.6205ms)\n  ✔ parallel\\test-async-local-storage-http-multiclients.js (4738.5198ms)\n  ✔ parallel\\test-async-local-storage-snapshot.js (4737.2896ms)\n▶ AsyncContextFrame (5523.243ms)\nℹ tests 23\nℹ suites 1\nℹ pass 21\nℹ fail 2\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 5545.2197\n\n✖ failing tests:\n\ntest at test\\parallel\\test-async-context-frame.mjs:48:5\n✖ async-hooks\\test-async-local-storage-args.js (4855.5555ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-args.js failed with exit code 3221225477\n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:888:9)\n      at async Promise.all (index 0)\n      at async Suite.run (node:internal/test_runner/test:1268:7)\n      at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }\n\ntest at test\\parallel\\test-async-context-frame.mjs:48:5\n✖ async-hooks\\test-async-local-storage-enter-with.js (4867.0779ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-enter-with.js failed with exit code 3221225477\n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:888:9)\n      at async Promise.all (index 5)\n      at async Suite.run (node:internal/test_runner/test:1268:7)\n      at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }"},"time":6060.198,"fullname":"parallel/test-async-context-frame","closingTestPoint":false}},{"testName":"parallel/test-runner-run-watch","timestamp":1725360895269,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win10-COMPILED_BY-vs2022 #29880","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win10-COMPILED_BY-vs2022/29880/","buildNumber":29880,"builtOn":"test-azure_msft-win10_vs2019-x64-1","nodeVersion":"23.0.0","commitHash":"0ae2ae1e52aa70bdc876a9e414bb324dfb645388","callStack":[{"upstreamBuild":65441,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":73848,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":61848,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54526"}],"tap":{"ok":false,"name":"parallel/test-runner-run-watch","id":876,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"timeout\n▶ test runner watch mode\n  ✖ should run tests repeatedly (6539.067ms)\n    AssertionError [ERR_ASSERTION]: The input did not match the regular expression /tests 1/. Input:\n    \n    '# Subtest: test.js\\n' +\n      'not ok 2 - test.js\\n' +\n      '  ---\\n' +\n      '  duration_ms: 1111.6943\\n' +\n      \"  location: 'c:\\\\\\\\workspace\\\\\\\\node-test-binary-windows-js-suites\\\\\\\\node\\\\\\\\test\\\\\\\\.tmp.685\\\\\\\\test.js:1:1'\\n\" +\n      \"  failureType: 'testCodeFailure'\\n\" +\n      '  exitCode: ~\\n' +\n      \"  signal: 'SIGTERM'\\n\" +\n      \"  error: 'test failed'\\n\" +\n      \"  code: 'ERR_TEST_FAILURE'\\n\" +\n      '  ...\\n' +\n      '# Subtest: test has ran\\n' +\n      'ok 2 - test has ran\\n' +\n      '  ---\\n' +\n      '  duration_ms: 18.7181\\n' +\n      '  ...\\n' +\n      '# Subtest: test.js\\n' +\n      'not ok 3 - test.js\\n' +\n      '  ---\\n' +\n      '  duration_ms: 913.4459\\n' +\n      \"  location: 'c:\\\\\\\\workspace\\\\\\\\node-test-binary-windows-js-suites\\\\\\\\node\\\\\\\\test\\\\\\\\.tmp.685\\\\\\\\test.js:1:1'\\n\" +\n      \"  failureType: 'testCodeFailure'\\n\" +\n      '  exitCode: ~\\n' +\n      \"  signal: 'SIGTERM'\\n\" +\n      \"  error: 'test failed'\\n\" +\n      \"  code: 'ERR_TEST_FAILURE'\\n\" +\n      '  ...\\n' +\n      '# Subtest: test has ran\\n' +\n      'ok 4 - test has ran\\n' +\n      '  ---\\n' +\n      '  duration_ms: 18.5125\\n' +\n      '  ...\\n' +\n      '1..4\\n' +\n      '# tests 4\\n' +\n      '# suites 0\\n' +\n      '# pass 2\\n' +\n      '# fail 2\\n' +\n      '# cancelled 0\\n' +\n      '# skipped 0\\n' +\n      '# todo 0\\n' +\n      '# duration_ms 1248.9526\\n'\n    \n        at testUpdate (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-run-watch.mjs:81:14)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async testWatch (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-run-watch.mjs:173:26)\n        at async TestContext.<anonymous> (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-run-watch.mjs:183:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Promise.all (index 0)\n        at async Suite.run (node:internal/test_runner/test:1268:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n      generatedMessage: true,\n      code: 'ERR_ASSERTION',\n      actual: \"# Subtest: test.js\\nnot ok 2 - test.js\\n  ---\\n  duration_ms: 1111.6943\\n  location: 'c:\\\\\\\\workspace\\\\\\\\node-test-binary-windows-js-suites\\\\\\\\node\\\\\\\\test\\\\\\\\.tmp.685\\\\\\\\test.js:1:1'\\n  failureType: 'testCodeFailure'\\n  exitCode: ~\\n  signal: 'SIGTERM'\\n  error: 'test failed'\\n  code: 'ERR_TEST_FAILURE'\\n...\",\n      expected: /tests 1/,\n      operator: 'match'\n    }"},"time":122254.121,"fullname":"parallel/test-runner-run-watch","closingTestPoint":false}},{"testName":"pummel/test-child-process-spawn-loop","timestamp":1725360895269,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win11-arm64-COMPILED_BY-vs2022-arm64 #29880","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/29880/","buildNumber":29880,"builtOn":"test-azure_msft-win11_vs2022-arm64-6","nodeVersion":"23.0.0","commitHash":"0ae2ae1e52aa70bdc876a9e414bb324dfb645388","callStack":[{"upstreamBuild":65441,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":73848,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":61848,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54526"}],"tap":{"ok":false,"name":"pummel/test-child-process-spawn-loop","id":962,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:126\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: Expected values to be strictly equal:\n\n0 !== 1024002\n\n    at ChildProcess.<anonymous> (d:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\pummel\\test-child-process-spawn-loop.js:48:12)\n    at ChildProcess.emit (node:events:520:28)\n    at maybeClose (node:internal/child_process:1104:16)\n    at ChildProcess._handle.onexit (node:internal/child_process:305:5) {\n  generatedMessage: true,\n  code: 'ERR_ASSERTION',\n  actual: 0,\n  expected: 1024002,\n  operator: 'strictEqual'\n}\n\nNode.js v23.0.0-pre"},"time":2477.544,"fullname":"pummel/test-child-process-spawn-loop","closingTestPoint":false}}]}